packages/mobile/src/dappkit/DappKitAccountScreen.tsx

```diff
@@ -119,42 +114,40 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     justifyContent: 'center',
     marginHorizontal: '15%',
+    marginVertical: '50%',
```

Comment:
I was surprised by this at first and did some more checks to be sure it works in all situations, 
turns out it starts clipping the content on smaller screens, like iPhone SE:

<img src="https://user-images.githubusercontent.com/57791/91580836-f5929780-e94d-11ea-9145-bad229045cc4.png" width="50%" />

Here's the full styles that make it work on all screen sizes, no matter what the content is, the trick is with `flexGrow: 1` and fixing the button taking the full height:

```jsx
const styles = StyleSheet.create({
  container: {
    flex: 1,
  },
  scrollContainer: {
    flexGrow: 1,
    alignItems: 'center',
    justifyContent: 'center',
    marginHorizontal: '15%',
  },
  header: {
    ...fontStyles.h1,
    alignItems: 'center',
    paddingBottom: 16,
  },
  share: {
    ...fontStyles.subSmall,
    color: colors.gray4,
    fontSize: 16,
    textAlign: 'center',
  },
  sectionDivider: {
    alignItems: 'center',
    width: 200,
  },
  sectionHeaderText: {
    ...fontStyles.sectionLabelNew,
    fontSize: 13,
    marginTop: 16,
  },
  bodyText: {
    ...fontStyles.regular,
    fontSize: 16,
    color: colors.gray4,
    textAlign: 'center',
  },
  button: {
    marginTop: 24,
  },
})
```

Result on iPhone SE and iPhone 11:

<img src="https://user-images.githubusercontent.com/57791/91580601-a2204980-e94d-11ea-9fe6-58bbc5ea6a46.png" width="50%" /><img src="https://user-images.githubusercontent.com/57791/91580577-9c2a6880-e94d-11ea-9b2a-0146f2a7f28c.png" width="50%" />